frontend/src/v5/ui/routes/dashboard/projects/federations/editFederationModal/editFederationModal.styles.ts

```diff
@@ -0,0 +1,102 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import styled from 'styled-components';
+import { FormDialogContent } from '@controls/modal/formModal/formDialog.styles';
+import { FormModal as FormModalBase } from '@controls/modal/formModal/formDialog.component';
+import { HeaderButtonsGroup } from '@/v5/ui/routes/dashboard/projects/containers/containers.styles';
+import BaseIncludeIcon from '@assets/icons/include_element.svg';
+import BaseRemoveIcon from '@assets/icons/remove_element.svg';
+
+export const FormModal = styled(FormModalBase)`
```

Comment:
I try to avoid customising the style of the components if I can. Often this means that the original component has a design flaw (is not flexible enough to support the common cases). 
Im not sure (havent attempt it), but do you think there's a way to support this and the other cases modifying the original `<FormModal />` without making it complex  ?

Reply:
The FormModal is not longer re-styled